src/server/game/Spells/Spell.cpp

```diff
@@ -8244,8 +8244,8 @@ void Spell::PrepareTriggersExecutedOnHit()
         case SPELLFAMILY_MAGE:
         {
             // Permafrost
-            if (m_spellInfo->SpellFamilyFlags[1] & 0x00001000 ||  m_spellInfo->SpellFamilyFlags[0] & 0x00100220)
-            m_preCastSpell = 68391;
+            // if (m_spellInfo->SpellFamilyFlags[1] & 0x00001000 ||  m_spellInfo->SpellFamilyFlags[0] & 0x00100220)
+            // m_preCastSpell = 68391;
```

Comment:
why commenting those lines instead of just removing them?

Reply:
We commented those lines just to be on the safe side as because its using flags we wouldn't know what else this effects (prob nothing) but we can always put them back in if needs be and then find a alternative fix for this

Reply:
ok

Reply:
Okay but we use version control, there's permanent history of what you remove. If you don't take them out now it'll likely never get removed.

Reply:
That is also true. On one hand it's easier to find as commented rather than in history (but still possible to find). On the other hand, the code is much cleaner if we remove such dead code instead of commenting it. I personally prefer to remove them in such cases, but I wouldn't be strict about it.